src/include/sof/debug/telemetry/telemetry.h

```diff
@@ -15,6 +15,12 @@
 /* disables calculating systick_averages */
 #define SOF_PERFORMANCE_MEASUREMENTS
 
+/* to be moved to Zephyr */
```

Comment:
ack - we should also prefix with ```SOF_TELEMETRY_```

Reply:
Moved to kconfig as SOF_TELEMETRY_PERFORMANCE_MEASUREMENTS